docs/sparkr.md

```diff
@@ -450,6 +450,42 @@ print(model.summaries)
 {% endhighlight %}
 </div>
 
+### Eager execution
```

Comment:
This will be the same level as "Applying User-Defined Function" where its parent topic is "SparkDataFrame Operations". So the heading is with three `#`.